api/swagger.yaml

```diff
@@ -800,6 +800,31 @@ paths:
         default:
           $ref: '#/responses/UnknownError'
 
+  /control/newDiscoveredAPIs:
```

Comment:
maybe externalDiscoveryAPIs?

Reply:
I really meant internal, I fixed it in another commit.
So, /control/internalDiscoveryAPIs.

This new /control/ endpoint is meant to be used by Gateways, in order to notify APIClarity of new APIs.
And in general, /control/ will group all the APIs of APIClarity "control-plane".

Reply:
cool :) will other control APIs also move under /control?
If it's meant for gateways, maybe we can add it explicitly to the path (e.g. /control/gateway/internalDiscoveryAPIs.)?
I'm less a fan of putting in the description who is the consumer of the API (maybe as example)